src/Config/Visits.php

```diff
@@ -4,6 +4,10 @@
 
 class Visits extends BaseConfig
 {
+	
+	//Enable tracking in all controllers using the post_controller_constructor event
+	public $trackAllPages = false;
```

Comment:
This needs to be `true` by default or current implementations will stop tracking when they upgrade. 
```suggestion
	public $trackAllPages = true;
```